src/frontend/posts_page/components/InputBlock.tsx

```diff
@@ -18,74 +24,31 @@ class InputBlock extends React.Component<{}, IInputBlockState> {
     }
 
     startEditor = () => {
-        document.getElementById("all-posts")!.style.height = "65vh";
-        this.setState({editing: !this.state.editing})
-    };
-
-    inputOnChange = (event: any) => {
-        this.setState({message: event.target.value});
-    };
-
-    sendPost = () => {
-
+        this.setState({editing: !this.state.editing});
```

Comment:
Why should it be `editing:true` ?

Reply:
because it means the user starts to edit the post, so editing is true. When not editing post or the input editor is closed, editing is false.